pydicom/tests/test_filereader.py

```diff
@@ -325,6 +325,24 @@ def test_specific_tags(self):
         ]
         assert expected == ctspecific_tags
 
+    def test_specific_tags_with_other_unkonwn_length_tags(self):
+        rtstruct_specific = dcmread(
```

Comment:
This is not related to the issue, codecov just pointed to an uncovered line in `filereader.py`, so I added a test to cover it (skipping sequences with undefined length if specific tags are defined and they are not in these tags).